test/integration/scheduler_perf/main_test.go

```diff
@@ -17,11 +17,20 @@ limitations under the License.
 package benchmark
 
 import (
+	"flag"
 	"testing"
 
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+func init() {
+	// default logging flags to more reasonable values for this test
+	flag.Set("v", "1")
```

Comment:
NOTE: because this happens before parsing, `-v=4` etc. still works, this just changes the default. klog doesn't support doing this in a cleaner way AFAIK, but setting eg `logtostderr` is pretty common in binaries throughout the project.